src/kirby/components/list/list.component.ts

```diff
@@ -130,7 +142,12 @@ export class ListComponent implements OnChanges {
   }
 
   onItemSelect(args: any) {
-    this.itemSelect.emit(this.listHelper.getSelectedItem(this.items, args));
+    this.selectedItem = this.listHelper.getSelectedItem(this.items, args);
+    this.items.forEach((item) => {
```

Comment:
I looks like you don't use the `selected` property anywhere. So you can revert this change.